sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ReceiverManager.cs

```diff
@@ -41,6 +41,9 @@ internal class ReceiverManager
             {
                 ReceiveMode = ProcessorOptions.ReceiveMode,
                 PrefetchCount = ProcessorOptions.PrefetchCount,
+                // Pass None for subqueue since the subqueue has already
+                // been taken into account when computing the EntityPath of the processor.
+                SubQueue = SubQueue.None
```

Comment:
This is not technically necessary (since None is the default), but I included it to help document why the option is not passed to the receiver.

Reply:
I find this helpful to follow the rationale.